
#### Integrations

##### ServiceNow v2

- Fixed an issue where the default value of table_name was not taken from the instance configuration.

#### Scripts

##### ServiceNowAddComment

- Fixed an issue where the default value of table_name was not taken from the instance configuration.
- Updated the Docker image to: *demisto/python3:3.10.13.89009*.
